application/database.py

```diff
@@ -4,11 +4,42 @@
 class DatabaseConnect():
     """Initializes connection to the database"""
     def __init__(self):
-        self.conn = connect(os.getenv('DATABASE_URL'))
+        # self.conn = connect(os.getenv('DATABASE_URL'))
```

Comment:
this approach looks good to me instead of exposing the credentials below.

Reply:
thanks i'll change it right back